LICENSES/vendor/gopkg.in/evanphx/json-patch.v4/LICENSE

```diff
@@ -0,0 +1,29 @@
+= vendor/gopkg.in/evanphx/json-patch.v4 licensed under: =
```

Comment:
This must've been added after I ran `./hack/update-kustomize.sh`. Let me know if you'd like me to remove from this PR.